app/views/shared/_footer.html.erb

```diff
@@ -55,7 +55,7 @@
             <h3 class="text-sm font-semibold leading-6 text-gray-900 dark:text-white">About us</h3>
             <ul role="list" class="mt-4 space-y-3">
               <li><%= link_to 'About', about_path, class: 'text-sm leading-6 text-gray-600 hover:text-gray-900 dark:text-gray-300 dark:hover:text-white' %></li>
-              <li><%= link_to 'Blog', ODIN_BLOG_URL, target: '_blank', rel: 'noreferrer', class: 'text-sm leading-6 text-gray-600 hover:text-gray-900 dark:text-gray-300 dark:hover:text-white' %></li>
+              <li><%= link_to 'Blog', '/blog', target: '_blank', rel: 'noreferrer', class: 'text-sm leading-6 text-gray-600 hover:text-gray-900 dark:text-gray-300 dark:hover:text-white' %></li>
```

Comment:
With a named route helper this would change to:

```suggestion
              <li><%= link_to 'Blog', blog_path, target: '_blank', rel: 'noreferrer', class: 'text-sm leading-6 text-gray-600 hover:text-gray-900 dark:text-gray-300 dark:hover:text-white' %></li>
```